### PR TITLE
UI changes have been done

### PR DESCRIPTION
**Pull request by rajkumar-telu**
Added spacing and line height to reduce the cluttering.
Direction of the top section has been changed to row in Mobile dimension. 
Added a class to the 'Continue Reading' instead of styling the button tag.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -54,7 +54,7 @@ <h3>Title One</h3>
                         Provident quia eos excepturi? dolore placeat voluptatem sapiente. Lorem ipsum dolor sit amet consectetur adipisicing elit.
                         Lorem ipsum dolor sit amet consectetur, adipisicing elit. Aliquam, excepturi.
                     </div>
-                    <button>Continue Reading</button>
+                    <button class="cont-read">Continue Reading</button>
```
Comment (sabharwal-garv) [inline index.html]: Use anchor tag here instead of button

File: index.html [hunk 2]
```diff
@@ -77,7 +77,7 @@ <h3>Title Two</h3>
                             eum enit facilis natus, errom ipsum dolor sit amet consectetur adipisicing elit. Aspernatur, perferendis.
                             Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis odit obcaecati ducimus dignissimos, fugiat ipsam!
                         </div>
-                        <button>Continue Reading</button>
+                        <button class="cont-read">Continue Reading</button>
```
Comment (sabharwal-garv) [inline index.html]: same

File: index.html [hunk 3]
```diff
@@ -100,7 +100,7 @@ <h3>Title Three</h3>
                             Provident quia eos excepturi? dolore placeat voluptatem sapiente. Lorem ipsum dolor sit amet consectetur adipisicing elit.
                             Lorem ipsum dolor sit amet consectetur, adipisicing elit. Aliquam, excepturi.
                         </div>
-                        <button>Continue Reading</button>
+                        <button class="cont-read">Continue Reading</button>
```
Comment (sabharwal-garv) [inline index.html]: same

**Comment by rajkumar-telu** [comment]
Changed button tag to anchor tag as requested.

**Review by sabharwal-garv — APPROVED** [review]

**Pull request merged**